uptane-standard.md

```diff
@@ -314,17 +325,17 @@ An attacker seeking to deny installation of updates may attempt one or more of t
 
 * *Drop-request attack:* Block network traffic outside or inside the vehicle.
 * *Slow retrieval attack:* Slow down network traffic, in the extreme case sending barely enough packets to avoid a timeout. Similar to a drop-request attack, except that both the sender and receiver of the traffic still think network traffic is unimpeded.
-* *Freeze attack:* Continue to send a previously known update to an ECU, even if a newer update exists.
-* *Partial bundle installation attack:* Install updates to some ECUs, but freeze updates on others.
+* *Freeze attack:* Continue to send a properly signed, but old, update bundle to an ECU, even if a newer update exists.
```

Comment:
*"to the ECUs", not "an ECU", since the bundle affects multiple ECUs.

I would also want to confirm with @JustinCappos or somebody equivalent that this change is okay. I don't know if making the freeze attack more general (freezing bundles, not individual ECU images) compromises later discussion regarding this attack. I do not remember and don't have time to re-evaluate.

Reply:
I will make the one change, but will not merge till getting the ok from Justin or others. Thanks for the input. 

Lois

Reply:
Ok. Its fixed.